tests/inline_formsets/tests.py

```diff
@@ -162,3 +162,17 @@ def test_zero_primary_key(self):
         PoemFormSet = inlineformset_factory(Poet, Poem, fields="__all__", extra=0)
         formset = PoemFormSet(None, instance=poet)
         self.assertEqual(len(formset.forms), 1)
+
+    def test_unsaved_fk_validate_unique(self):
+        poet = Poet(name='unsaved')
+        PoemFormSet = inlineformset_factory(Poet, Poem, fields=['name'])
+        data = {
+            'poem_set-TOTAL_FORMS': '2',
+            'poem_set-INITIAL_FORMS': '0',
+            'poem_set-MAX_NUM_FORMS': '2',
+            'poem_set-0-name': 'Poem',
+            'poem_set-1-name': 'Poem',
+        }
+        formset = PoemFormSet(data, instance=poet)
+        self.assertFalse(formset.is_valid())
+        self.assertEqual(formset.errors, [{}, {'__all__': [formset.get_form_error()]}])
```

Comment:
I'm not sure why `get_form_error()` is named as it is. I would find the test more readable if you replaced the method call with the "Please correct the duplicate values below." string, but whatever you think.